websites/B/Battlefield Portal/dist/metadata.json

```diff
@@ -47,4 +48,4 @@
         "value": true
       }
     ]
-  }
\ No newline at end of file
+  }
```

Comment:
Bing bong missed this also bump version

Reply:
Yeah I knew I forgot _something_ in my sleep. Will fix ASAP

Reply:
Bumped 👊 and ignore the typo plz